plugins/mac-avcapture/data/locale/en-US.ini

```diff
@@ -11,3 +11,4 @@ VideoRange.Partial="Partial"
 VideoRange.Full="Full"
 Auto="Auto"
 Unknown="Unknown ($1)"
+EnableAudio="Enable audio (if supported by device)"
```

Comment:
The parenthetical probably doesn't need to be a parenthetical, but is also probably fine either way.